dojo/models.py

```diff
@@ -838,6 +838,8 @@ class Tool_Configuration(models.Model):
                                                 'Username/Password'),
                                                ('SSH', 'SSH')),
                                            null=True, blank=True)
+    extras = models.CharField(max_length=50, null=True, blank=True, help_text="Additional definitions that will be "
```

Comment:
just a thought, but might 50 be too small for some future tools? maybe use 255?

Reply:
Done